android/anc/src/test/java/org/smartregister/fhircore/anc/util/AncOverviewConfigurationTest.kt

```diff
@@ -52,7 +53,8 @@ class AncOverviewConfigurationTest : RobolectricTest() {
         BPSFilter,
         BPDSFilter,
         pulseRateFilter,
-        bloodGlucoseFilter
+        bloodGlucoseFilter,
+        bmiFilter
```

Comment:
Add name to function parameters to improve readabiity

Reply:
done